README.md

```diff
@@ -0,0 +1,19 @@
+# Small python docker image
+
+Based on alpine.
+
+See https://github.com/gliderlabs/docker-alpine
+
+## Usage
+
+Just derive from the image in `FROM` and the current directory will be added to the image and pip packages will be installed.
```

Comment:
I think it should say that the current directory will be added as `/app`. An important detail IMHO.